src/test/scala/nl.knaw.dans.easy.deposit/DataFilesSpec.scala

```diff
@@ -27,7 +27,6 @@ import nl.knaw.dans.easy.deposit.Errors.NoSuchFileInDepositException
 import nl.knaw.dans.easy.deposit.docs.FileInfo
 import nl.knaw.dans.lib.error._
 
-import scala.collection.JavaConverters._
 import scala.util.{ Failure, Success }
 
 class DataFilesSpec extends TestSupportFixture {
```

Comment:
Not entirely sure, but looking through this class, I don't see any tests for the backwards compatibility cases...